README.md

```diff
@@ -96,17 +96,18 @@ evaluation = MTEB(task_categories=['S2S']) # Only select sentence2sentence datas
 * by their languages
 
 ```python
+import mteb
 evaluation = MTEB(task_langs=["en", "de"]) # Only select datasets which are "en", "de" or "en-de"
 ```
 
-You can also specify which languages to load for multilingual/crosslingual tasks like below:
+You can also specify which languages to load for multilingual/cross-lingual tasks like below:
 
 ```python
 from mteb.tasks import AmazonReviewsClassification, BUCCBitextMining
 
 evaluation = MTEB(tasks=[
-        AmazonReviewsClassification(langs=["en", "fr"]) # Only load "en" and "fr" subsets of Amazon Reviews
-        BUCCBitextMining(langs=["de-en"]), # Only load "de-en" subset of BUCC
+        AmazonReviewsClassification(hf_subsets=["en", "fr"]) # Only load "en" and "fr" subsets of Amazon Reviews
+        BUCCBitextMining(hf_subsets=["de-en"]), # Only load "de-en" subset of BUCC
```

Comment:
Sorry if I'm missing something obvious, but why is this change needed? 🤔 Can these values ever not be languages?
(changing this breaks existing MTEB scripts)

Reply:
Yea I believe in oscar there is e.g. filtered_dan and unfiltered_dan. They are however often languages, but can be anything.